game-core/src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -861,9 +859,9 @@ private void doScrambling() {
         // Validate players have enough fuel to move there and back
         final Map<PlayerId, ResourceCollection> playerFuelCost = new HashMap<>();
         for (final Entry<Territory, Collection<Unit>> entry : toScramble.entrySet()) {
-          final Map<PlayerId, ResourceCollection> map =
+          final var map =
               Route.getScrambleFuelCostCharge(entry.getValue(), entry.getKey(), to, data);
-          for (final Entry<PlayerId, ResourceCollection> playerAndCost : map.entrySet()) {
+          for (final var playerAndCost : map.entrySet()) {
```

Comment:
My first reaction is this would be better named: `playerAndCostEntry` so we can keep some information on the typing. I'm not sure if that is more what I'm used to or actually helpful, but perhaps worth considering.